scrapy/commands/fetch.py

```diff
@@ -28,9 +29,10 @@ def add_options(self, parser):
             help="print response HTTP headers instead of body")
 
     def _print_headers(self, headers, prefix):
+        prefix = prefix.encode()
```

Comment:
I'd prefer using byte constants for `prefix` values - using `.encode()` raises questions because it uses `sys.getdefaultencoding()` which can be different from terminal encoding, so generally this code is not correct. It is not a bug here because `prefix` values are just two constants (> and <).